multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minisatip
+PKG_VERSION:=0.7.15
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO := git
+PKG_SOURCE_URL := https://github.com/catalinii/minisatip.git
+PKG_SOURCE_VERSION := fdbf44fc48392b0735e0bf17b31aaff8f06740d0
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/minisatip
+  TITLE:=SATIP server
+  URL:=https://github.com/catalinii/minisatip
+  SECTION:=multimedia
+  CATEGORY:=Multimedia
+endef
+
+CONFIGURE_ARGS += --with-linux-headers=$(LINUX_DIR)
```

Comment:
Is this configure flag really needed for compilation?  Seems like the project README has no mention of it and I cannot find refs to in the configure.ac either.